recipes/pyproject-fmt-rust/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+c_stdlib_version:
+  - "10.12"  # [osx and x86]
```

```suggestion
c_stdlib_version:  # [osx and x86_64]
  - "10.13"  # [osx and x86]
```
You will need at least 10.13 or it will break on the feedstock

---

[936a798](https://github.com/conda-forge/staged-recipes/pull/26329/commits/936a7988f3c896d2d190b7e914b62bc75b8dd569)